rules/rules-overridden-azure/technology-usage/embedded-framework.windup.xml

```diff
@@ -19,6 +19,31 @@
         <overrideRules>true</overrideRules>
     </metadata>
     <rules>
+        <rule id="embedded-framework-09000">
+            <when>
+                <or>
+                    <file filename="feign-{*}.jar"/>
+                    <project>
+                        <artifact groupId="com.netflix.feign" artifactId="{*}"/>
+                    </project>
+                    <project>
+                        <artifact groupId="org.springframework.cloud" artifactId="spring-cloud-starter-feign"/>
+                    </project>
+                </or>
+            </when>
+            <perform>
+                <hint  title="Embedded framework - Feign" category-id="information" effort="0">
+                    <message>
+                        The application embeds the Feign framework. If you are migrating to Azure Spring Apps, it is recommended to use Spring Cloud OpenFeign instead (Spring Cloud Feign has been deprecated).
```

Comment:
Let's leave it like that for now, but that's the typical case where we should have the same rule for Spring Apps with a dedicated message